<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMode</class>
 <widget class="QDialog" name="AdminMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>1311</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBackButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="FindEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>165</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>查 找</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CreateButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>CreateButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>FindEdit_2</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>GoBackButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
